newbank/client/ExampleClient.java

```diff
@@ -42,15 +42,7 @@ public void run() {
 			try {
 				while(true) {
 					String command = userInput.readLine();
-					/* the 10 refers to the menu number for Exiting (value needs to be adjusted when more items are
-					added to the menu!. */
-					if(command.equals("10")){
-						System.out.println("Thank you and have a nice day!");
-						System.exit(0);
-					}
-					else{
-						bankServerOut.println(command);
-					}
```

Comment:
In the case when the user enters 10, NewBankServer turns off, and ExampleClient continues to work and returns nulls